nc/include/graphics/Environment.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "Assets.h"
+
+namespace nc
+{
+    const std::string SkyboxMeshPath = "skybox.nca";
+
+    struct EnvironmentData
```

Comment:
I don't think this is part of the API. If you structure Environment like the other user-facing systems, I think you can keep everything hidden:

// here in include/
class Environment : public SystemBase
{
    public:
        virtual void SetCubeMap(const CubeMapFaces&) = 0;
};

// somewhere in source/
class EnvironmentImpl : public Environment
{
    ...
};

Reply:
ahh my formatting